odd-platform-ui/src/components/Overview/Overview.tsx

```diff
@@ -162,12 +162,7 @@ const Overview: React.FC<OverviewProps> = ({
             </S.InfoBarItem>
           </Grid>
           {identity?.owner ? (
-            <Grid
-              container
-              wrap="nowrap"
-              justifyContent="space-between"
-              sx={{ mt: 3.5, mx: 'auto' }}
-            >
+            <S.DataEntityContainer container wrap="nowrap">
```

Comment:
If you have created styled component for this, could you move flexWrap property to styles too?